robosat/log.py

```diff
@@ -0,0 +1,26 @@
+"""Log facilitator
+"""
+
+import os
+
+
+class Log:
+
+    """Create a log instance on a log file
+    """
+
+    def __init__(self, path, stdout=True):
```

Comment:
Instead of a boolean what about letting the user pass `sys.stdout` or `sys.stderr` here? Or `None` for don't print.